assets/xml/textures/nes_font_static.xml

```diff
@@ -0,0 +1,143 @@
+        <Texture Name="gMsgCharUppercaseB" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseC" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseD" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseE" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseF" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseG" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseH" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseI" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseJ" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseK" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseL" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseM" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseN" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseO" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseP" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseQ" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseR" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseS" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseT" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseU" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseV" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseW" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseX" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseY" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUppercaseZ" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLeftSquareBracket" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharYenSign" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharRightSquareBracket" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharCaret" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharUnderscore" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharGraveAccent" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLowercase_a" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLowercase_b" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLowercase_c" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLowercase_d" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLowercase_e" Format="i4" Width="16" Height="16"/>
```

Comment:
I see why you used an underscore, but im wondering if theres a way to do it without breaking our naming style. Other than that these seem fine to me

Reply:
oh actually, all of these need an OutName so that the pngs have snake_case names without all the extra variable name stuff. can look at gameplay keep as an example